calcom/cal.com
# feat: removed the confirm step in creating bookings

## What does this PR do?

Removed the confirm step changes in this PR because as I see it, once the URL params are passed and checked in the useEffect, the booking state is changed, where I've replaced it with the handleEventBooking();

- Fixes #18307 (GitHub issue number)
- Fixes CAL-4943 (Linear issue number - should be visible at the bottom of the GitHub issue description)

## Mandatory Tasks (DO NOT REMOVE)

- [N/A] I have updated the developer docs in /docs if this PR makes changes that would require a [documentation change](https://cal.com/docs). If N/A, write N/A here and check the checkbox.

LOOM LINK: https://www.loom.com/share/2967cdb902fb4945908e17c79bb379b8?sid=88459728-1b9e-46e9-af1a-9168cd8cc0cc

## How should this be tested?

You can start by creating event types with varying complexity and check if any URL params are conflicting this direct booking without confirmation.

No ENV vars required to test.
